src/Surfnet/ServiceProviderDashboard/Infrastructure/DashboardBundle/Controller/EntityCreateController.php

```diff
@@ -145,9 +145,9 @@ public function createAction(Request $request, $serviceId, $targetEnvironment, $
 
         $service = $this->authorizationService->changeActiveService($serviceId);
 
-        if ($type === Entity::TYPE_OPENID_CONNECT && !$this->authorizationService->isOidcCreateEntityAllowed()) {
+        if ($type === Entity::TYPE_OPENID_CONNECT) {
             throw $this->createAccessDeniedException(
```

Comment:
I think a 404 is more in place then a 403.

Reply:
I'd say I have to agree on that